pkg/virt-api/webhooks/mutating-webhook/mutators/clone-create-mutator_test.go

```diff
@@ -17,43 +15,81 @@ import (
 
 	"kubevirt.io/api/clone"
 	clonev1alpha1 "kubevirt.io/api/clone/v1alpha1"
-	"kubevirt.io/client-go/kubecli"
 )
 
 var _ = Describe("Clone mutating webhook", func() {
+	const testSourceVirtualMachineName = "test-source-vm"
+
 	It("Target should be auto generated if missing", func() {
-		vmClone := kubecli.NewMinimalCloneWithNS("testclone", util.NamespaceTestDefault)
-		vmClone.Spec.Source = &k8sv1.TypedLocalObjectReference{
-			APIGroup: pointer.String(clone.GroupName),
-			Kind:     "VirtualMachine",
-			Name:     "test-source-vm",
-		}
+		vmClone := newVirtualMachineClone(
+			withVirtualMachineSource(testSourceVirtualMachineName),
+		)
 
 		cloneSpec := mutate(vmClone)
 		Expect(cloneSpec.Target).ShouldNot(BeNil())
 		Expect(cloneSpec.Target.Name).ShouldNot(BeEmpty())
 	})
 
 	It("Target name should be auto generated if missing", func() {
-		vmClone := kubecli.NewMinimalCloneWithNS("testclone", util.NamespaceTestDefault)
-		vmClone.Spec.Source = &k8sv1.TypedLocalObjectReference{
-			APIGroup: pointer.String(clone.GroupName),
-			Kind:     "VirtualMachine",
-			Name:     "test-source-vm",
-		}
+		vmClone := newVirtualMachineClone(
+			withVirtualMachineSource(testSourceVirtualMachineName),
+			withVirtualMachineTarget(""),
+		)
 
-		vmClone.Spec.Target = &k8sv1.TypedLocalObjectReference{
-			APIGroup: pointer.String(clone.GroupName),
-			Kind:     "VirtualMachine",
-			Name:     "",
-		}
 		cloneSpec := mutate(vmClone)
 		Expect(cloneSpec.Target).ShouldNot(BeNil())
 		Expect(cloneSpec.Target.Name).ShouldNot(BeEmpty())
 	})
 
 })
 
+type option func(vmClone *clonev1alpha1.VirtualMachineClone)
+
+func newVirtualMachineClone(options ...option) *clonev1alpha1.VirtualMachineClone {
+	const (
+		testNamespace               = "kubevirt-test-default"
+		testVirtualMachineCloneName = "testclone"
+	)
```

Comment:
nit: I do not know if the linter complains or not, but no additional information is added when these consts are used. It is very clear what they refer to, because they appear with the member name.

Reply:
Done.